qt/aqt/editor.py

```diff
@@ -1621,11 +1609,12 @@ def _maybe_add_image_menu(self, menu: QMenu) -> None:
             a = menu.addAction(tr.editing_copy_image())
             qconnect(a.triggered, self.on_copy_image)
 
+            # inspired by Image Occlusion Enhanced add-on
```

Comment:
Sorry, my bad, I think I should have expressed myself better: I just meant leaving a small note in the PR in reference to the add-on :) I think including credits like this in the code is a nice gesture, but if we do it for every change inspired by add-ons, especially very small and trivial ones, the codebase would get very noisy very quickly. Perhaps something like this would _maybe_ make sense for much larger pieces of code (like full modules/packages). Though there I think that add-on authors would moreso be happy to just have a small footnote in changelogs, if the contribution/inspiration is large enough